src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrDependencyTracker.cs

```diff
@@ -56,32 +56,32 @@ public class VmrDependencyTracker : IVmrDependencyTracker
     public VmrDependencyVersion? GetDependencyVersion(SourceMapping mapping)
         => _sourceManifest.GetVersion(mapping.Name);
 
-    public void UpdateDependencyVersion(SourceMapping mapping, VmrDependencyVersion version)
+    public void UpdateDependencyVersion(VmrDependencyUpdate update)
     {
         // TODO: https://github.com/dotnet/source-build/issues/2250
-        if (version.PackageVersion is null)
+        if (update.TargetVersion is null)
         {
-            version = version with { PackageVersion = DefaultVersion };
+            update = update with { TargetVersion = DefaultVersion };
```

Comment:
Nit: `DefaultVersion` is only used here. We can make it only visible inside `UpdateDependencyVersion` method.

Reply:
I'd rather keep it up and more visible as it's quite important